examples/log/bipedal_walk.log

```diff
@@ -1,98 +1,119 @@
+  12  6.50785e+03  2.09132e+07  2.49071e+03  1.00000e-08  1.00000e-08   1.0000     1
+  13  6.39300e+03  2.09359e+05  6.42785e+02  1.00000e-09  1.00000e-09   1.0000     1
+  14  6.31464e+03  6.75745e+04  4.39058e+02  1.00000e-09  1.00000e-09   0.5000     1
+  15  6.18646e+03  1.64086e+04  2.57202e+02  1.00000e-09  1.00000e-09   1.0000     1
+  16  6.18644e+03  2.23188e+01  1.61587e+00  1.00000e-09  1.00000e-09   0.0625     1
+  17  6.18611e+03  2.54375e+01  1.49405e+00  1.00000e-09  1.00000e-09   0.2500     1
+  18  6.18609e+03  1.40771e+01  8.40725e-01  1.00000e-09  1.00000e-09   0.1250     1
+  19  6.18579e+03  1.08440e+01  5.88493e-01  1.00000e-09  1.00000e-09   1.0000     1
 iter 	 cost 	      stop 	    grad 	  xreg 	      ureg 	 step 	 feas
-  10  5.91708e+06  3.31440e+13  1.41165e+07  1.00000e-02  1.00000e-02   0.1250     1
-  11  5.12334e+06  6.25404e+12  1.17816e+07  1.00000e-02  1.00000e-02   0.1250     1
-  12  4.47999e+06  1.15757e+13  1.01993e+07  1.00000e-02  1.00000e-02   0.1250     1
-  13  4.23743e+06  2.48241e+13  8.91777e+06  1.00000e-02  1.00000e-02   0.2500     1
-  14  3.47031e+06  9.37477e+13  8.43998e+06  1.00000e-02  1.00000e-02   0.5000     1
-  15  2.27630e+06  1.71309e+14  6.91393e+06  1.00000e-02  1.00000e-02   0.5000     1
-  16  7.21762e+04  1.36749e+14  4.53210e+06  1.00000e-03  1.00000e-03   1.0000     1
-  17  5.35070e+04  3.93018e+12  1.32586e+05  1.00000e-03  1.00000e-03   0.5000     1
-  18  4.07228e+04  2.87062e+12  9.61531e+04  1.00000e-04  1.00000e-04   1.0000     1
-  19  3.49968e+04  1.82896e+12  7.17532e+04  1.00000e-05  1.00000e-05   1.0000     1
+  20  6.18574e+03  2.88529e+00  1.01709e-01  1.00000e-09  1.00000e-09   1.0000     1
+  21  6.18574e+03  4.20303e-03  1.88796e-05  1.00000e-09  1.00000e-09   1.0000     1
+  22  6.18574e+03  6.52452e-04  1.85541e-07  1.00000e-09  1.00000e-09   1.0000     1
+  23  6.18574e+03  1.51222e-06  6.38506e-09  1.00000e-09  1.00000e-09   1.0000     1
+  24  6.18574e+03  2.20327e-08  1.40621e-09  1.00000e-09  1.00000e-09   1.0000     1
+iter 	 cost 	      stop 	    grad 	  xreg 	      ureg 	 step 	 feas
+   0  8.16841e+08  4.46091e+12  1.63057e+09  1.00000e+00  1.00000e+00   0.0020     0
+   1  8.16841e+08  4.30532e+12  1.62891e+09  1.00000e+01  1.00000e+01   0.0020     0
+   2  6.46220e+07  4.40688e+12  1.62520e+09  1.00000e+00  1.00000e+00   1.0000     1
+   3  3.21119e+07  7.21526e+14  1.28839e+08  1.00000e+00  1.00000e+00   0.5000     1
+   4  3.08284e+07  6.45283e+14  6.40505e+07  1.00000e+00  1.00000e+00   0.0625     1
+   5  2.87518e+07  7.19990e+14  6.14803e+07  1.00000e+00  1.00000e+00   0.1250     1
+   6  1.52045e+07  8.06352e+14  5.73367e+07  1.00000e-01  1.00000e-01   1.0000     1
+   7  1.44841e+07  5.61935e+14  3.03400e+07  1.00000e-01  1.00000e-01   0.2500     1
+   8  1.11486e+07  6.71913e+14  2.89160e+07  1.00000e-01  1.00000e-01   0.2500     1
+   9  6.03658e+06  5.76574e+14  *** SOLVE walking ***
+*** SOLVE walking ***
```

Comment:
This is a new change in output order / not flushing properly that I haven't seen before